src/LiveDevelopment/MultiBrowserImpl/documents/LiveCSSDocument.js

```diff
@@ -64,6 +78,8 @@ define(function LiveCSSDocumentModule(require, exports, module) {
     var LiveCSSDocument = function LiveCSSDocument(protocol, urlResolver, doc, editor, roots) {
         LiveDocument.apply(this, arguments);
 
+        this.roots = normalizeRoots(this.roots);
```

Comment:
I want to leave it as `this.roots` in case the base constructor messes with it in any way, after the call on 79.